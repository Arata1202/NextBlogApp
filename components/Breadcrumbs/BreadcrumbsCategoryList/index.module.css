.tags {
  display: flex;
  flex-wrap: wrap;
  gap: 8px;
}
