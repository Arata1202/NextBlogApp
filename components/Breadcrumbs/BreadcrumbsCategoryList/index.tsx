import { Category } from '@/libs/microcms';
import BreadcrumbsCategoryListItem from '../BreadcrumbsCategoryListItem';
import styles from './index.module.css';

type Props = {
  categories?: Category[];
  hasLink?: boolean;
};

export default function PanTagList({ categories, hasLink = true }: Props) {
  if (!categories) {
    return null;
  }
  return (
    <ul className={styles.tags}>
      {categories.map((category) => (
        <li key={category.id}>
          <BreadcrumbsCategoryListItem category={category} hasLink={hasLink} />
        </li>
      ))}
    </ul>
  );
}
