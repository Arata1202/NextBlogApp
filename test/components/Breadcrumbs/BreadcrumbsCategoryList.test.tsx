import { render } from '@testing-library/react';
import BreadcrumbsCategoryList from '@/components/Breadcrumbs/BreadcrumbsCategoryList';

describe('BreadcrumbsCategoryList', () => {
  test('BreadcrumbsCategoryList)', () => {
    const { asFragment } = render(<BreadcrumbsCategoryList />);
    expect(asFragment()).toMatchSnapshot();
  });
});
