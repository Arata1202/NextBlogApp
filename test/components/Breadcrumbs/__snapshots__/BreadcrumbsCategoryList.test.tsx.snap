// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BreadcrumbsCategoryList BreadcrumbsCategoryList) 1`] = `<DocumentFragment />`;
